charts/kafka-operator/templates/operator-service.yaml

```
@@ -28,6 +28,7 @@ spec:
   ports:
   - name: https
     port: 443
+    targetPort: {{ default 443 .Values.webhook.serverPort }}
```

```suggestion
    targetPort: {{ .Values.webhook.serverPort | default 443 }}
```
I would use it this way to stay in line with how we use it in the rest of the project

---

Hello @hi-im-aren, thank you for the quick review. 
I've updated the code according to the recommendations.